pkg/account/handler.go

```diff
@@ -173,48 +169,18 @@ func (h *AccountHandler) LoginAccount(c echo.Context) error {
 		return c.JSON(http.StatusBadRequest, api.Return("Wrong Password", nil))
 	}
 
-	account.Token, _ = account.GenerateToken()
-	cookie := http.Cookie{
-		Name:     "token",
-		Value:    account.Token,
-		Expires:  time.Now().Add(7 * 24 * time.Hour),
-		Path:     "/api",
-		SameSite: http.SameSiteNoneMode,
-		Secure:   true,
-	}
-	c.SetCookie(&cookie)
+	token, err := account.GenerateToken()
 
-	if result := db.Model(&Account{}).Where("id = ?", account.ID).Update("token", account.Token); result.Error != nil {
-		return c.JSON(http.StatusBadRequest, api.Return("DB error", result.Error))
+	if err != nil {
+		return c.JSON(http.StatusInternalServerError, api.Return("fail to generete token", err))
 	}
 
 	return c.JSON(http.StatusOK, api.Return("Logged in", echo.Map{
-		"account":      account,
-		"cookie_token": account.Token,
+		"account": account,
+		"token":   token,
 	}))
 }
 
-// @Summary logout using cookie
-// @Description
-// @Tags Account
-// @Produce json
-// @Success 200 {string} api.ReturnedData{data=nil}
-// @Failure 400 {string} api.ReturnedData{data=nil}
-// @Router /account/logout [POST]
-func (h *AccountHandler) LogoutAccount(c echo.Context) error {
```

Comment:
Account cannot be logged out until another user is logged in?

Reply:
If you want to logout, simply drop the token in the frontend